soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4271,7 +4377,28 @@ CustomMessageEntry Randomizer::GetMerchantMessage(RandomizerInf randomizerInf, u
     return messageEntry;
 }
 
-const char* mapGetItemHints[3][2] = {
+CustomMessageEntry Randomizer::GetCursedSkullMessage(s16 params) {
+    CustomMessageEntry messageEntry = CustomMessageManager::Instance->RetrieveMessage(Randomizer::randoMiscHintsTableID, TEXT_CURSED_SKULLTULA_PEOPLE);
+    RandomizerCheck rc = GetCheckFromActor(ACTOR_EN_SSH, SCENE_KINSUTA, params);
+    RandomizerGet itemGet = this->itemLocations[rc].rgID;
+    std::vector<std::string> itemName;
+    if (itemGet == RG_ICE_TRAP) {
+        itemGet = this->itemLocations[rc].fakeRgID;
+        itemName = {
+            std::string(this->itemLocations[rc].trickName),
+            std::string(this->itemLocations[rc].trickName),
+            std::string(this->itemLocations[rc].trickName)
+        };
+    } else {
+        itemName = EnumToSpoilerfileGetName[itemGet];
+    }
+
+    CustomMessageManager::ReplaceStringInMessage(messageEntry, "{{params}}", std::to_string(params*10));
+    CustomMessageManager::ReplaceStringInMessage(messageEntry, "{{check}}", itemName[0], itemName[1], itemName[2]);
+    return messageEntry;
+}
+
+std::vector<std::vector<const char*>> mapGetItemHints = {
```

Comment:
A merge conflict probably reverted this from a const char*
```suggestion
const char* mapGetItemHints[3][2] = {
```